addons/project/views/project_views.xml

```diff
@@ -899,7 +899,7 @@
                                 </tree>
                             </field>
                         </page>
-                        <page name="recurrence" string="Recurrence" attrs="{'invisible': [('recurring_task', '=', False)]}">
+                        <page name="recurrence" string="Recurrence" attrs="{'invisible': ['|', ('recurring_task', '=', False), ('allow_recurring_tasks', '=', False)]}">
```

Comment:
Is this really necessary?
As we have a mechanism on the write that updates all tasks accordingly [here](https://github.com/odoo/odoo/blob/4e2adc9b1c749d7181941c7acd1e10afb9008c93/addons/project/models/project.py#L537).

Reply:
Hello @IT-Ideas  If recurrence (in configuration > settings) is false and 'recurring_task' is true in the import file, the Recurrence tab appears. [See here](https://drive.google.com/file/d/1-TM6ZCV7tbaCTh2L6fbsS9OIZDP0QCtn/view?usp=sharing) Thanks

Reply:
I think that we should prevent recurring_task=True in the import if the project setting is not set to True.
Don't you agree?

Reply:
@IT-Ideas 👍  Thanks